### ISSUE TITLE
Missing debug symbols for Windows releases

### ISSUE THREAD

**Issue by iforapsy** [ossec/ossec-hids#1043]
I cannot find the debug symbol files for ossec-agent.exe. I looked through the [Releases page](https://github.com/ossec/ossec-hids/releases) and the [Downloads page](https://ossec.github.io/downloads.html). There should be a debug symbol download link next to every exe download link. The symbols are necessary for troubleshooting OSSEC issues.

**Comment by ddpbsd** [comment]
By "there should be a..." do you mean "I think you should provide this stuff for me?"

**Comment by iforapsy** [comment]
Yes, please. If you could provide symbols for the official releases, that would be very helpful.

**Comment by ddpbsd** [comment]
Sure, do you happen to know the commands to get the debugging symbols?
